src/EventStore.Core/Services/Storage/EpochManager/EpochManager.cs

```diff
@@ -216,6 +221,9 @@ public class EpochManager : IEpochManager {
 
 				epoch = sysRec.GetEpochRecord();
 				return epoch.EpochNumber == epochNumber && epoch.EpochId == epochId;
+			} catch(Exception ex) when (ex is InvalidReadException || ex is UnableToReadPastEndOfStreamException) {
+				Log.Warning("Failed to read epoch {epochNumber} at {epochPosition}. Exception: {ex}", epochNumber, epochPosition, ex);
```

Comment:
should this be a warning or information?

Reply:
It should be Information because it's not likely to cause issues, I've updated it